style.css

```diff
@@ -3,16 +3,22 @@
     padding: 0;
 }
 
-.ques-header {
+.quiz-wrapper {
     background: var(--c-lightblue);
 }
 
 .ques-wrapper {
     width: 35rem;
+    background: var(--c-white);
+}
+
+.material-icons {
+    font-size: 140%;
```

Comment:
avoid using size greater than 100% it will effect when code incresed

Reply:
thanks. i have replaced it with rem.